storage/zynq7000-flash/tests/flashdrv.c

```diff
@@ -0,0 +1,399 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Nor Flash driver tests for zynq-7000
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <flashdrv.h>
+#include <flashcfg.h>
+
+#include "unity_fixture.h"
+
+#define SIZE_DEFAULT_BUFF    0x10000
+#define VALUE_DEFAULT_SAMPLE 0xa5
+
+struct {
+	uint8_t *txBuff;
+	uint8_t *rxBuff;
+	size_t buffSz;
+
+	flash_info_t info;
+} common;
+
+
+/* Auxiliary functions */
+
+static inline void test_initBuffs(size_t sz)
+{
+	int i;
+
+	TEST_ASSERT_GREATER_OR_EQUAL_INT32(sz, common.buffSz);
+
+	for (i = 0; i < sz; i++)
+		common.txBuff[i] = i % 0xff;
```

Comment:
% 0xff is redundant - this operation is performed implicitly anyway

Reply:
Done